src/lumen_anndata/source.py

```diff
@@ -576,5 +628,86 @@ def create_sql_expr_source(
         return source
 
     def to_spec(self, context: dict[str, Any] | None = None) -> dict[str, Any]:
-        # TODO: temporarily disable this until Lumen internal supports unserializable objects
-        return {}
+        filename = self._adata_store.filename
+        if filename is None:
+            filename = tempfile.mktemp(suffix=".h5ad")
```

Comment:
Maybe use instead [NamedTemporaryFile()](https://docs.python.org/3/library/tempfile.html#tempfile.NamedTemporaryFile), passing it the delete=False parameter.

See [deprecation warning](https://docs.python.org/3/library/tempfile.html#:~:text=Warning%20Use%20of,delete%3DFalse%20parameter%3A)

Reply:
```
with tempfile.NamedTemporaryFile(suffix=".h5ad", delete=False) as tmp:
    filename = tmp.name
```

etc